db/migrate/20220125182524_create_tasks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.text :description
```

Comment:
Do columns get created with `null: true` by default? If so, should we be allowing tasks to be saved without a description? Ignore me if this isn't the case 😅

Reply:
You are right, it would have to explicitly state `null: false`

Reply:
I am not sure if I understand what do you mean by that 🤔 
In the Task model I  used this validator: `validates :description, presence: true` so a Task won't be save in the database if the description is missing. I thought that is enough.
But if there is another way to validate when create the tables, I would like to know.

Reply:
No worries! 👍🏼  So using that validator will _only_ validate tasks that are created through the app, which in this case would be through POST requests to your backend. However, it won't stop anyone from creating "null" tasks directly from the database using SQL. Currently as it stands, anyone can connect to your database and do this:

<img width="981" alt="Screen Shot 2022-01-26 at 8 21 21 AM" src="https://user-images.githubusercontent.com/6445410/151203193-b7775dcf-476d-414f-b44e-84dcc17ca951.png">

Which makes sense, given the output of describing the `Tasks` table:

<img width="963" alt="Screen Shot 2022-01-26 at 8 20 35 AM" src="https://user-images.githubusercontent.com/6445410/151203280-ca3325a7-c54b-4f3d-a465-1b94a5820621.png">

We can see that the `Nullable` value for `description`, is empty, meaning that `description` can be saved with null values. Therefore for any column that we wouldn't want to be saved as null, we'd need to explicitly state that using `null: false` in the migration.

Reply:
Thank you for explaining.